.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Problems and issues with code or docs
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+<!--
+
+Welcome to Apache Airflow!  For a smooth issue process, try to answer the following questions.
+Don't worry if they're not all applicable; just try to include what you can :-)
+
+If you need to include code snippets or logs, please put them in fenced code
+blocks.  If they're super-long, please use the details tag like
+<details><summary>super-long log</summary> lots of stuff </details>
+
+Please delete these comment blocks before submitting the issue.
+
+-->
+
+<!--
+
+IMPORTANT!!!
+
+Please complete the next sections or the issue will be closed.
+This questions are the first thing we need to know to understand the context.
+
+-->
+
+**Apache Airflow version**:
+
+**What happened**:
+
+<!-- (please include exact error messages if you can) -->
+
+**What you expected to happen**:
+
+<!-- What do you think went wrong? -->
+
+**How to reproduce it**:
+<!---
+
+As minimally and precisely as possible. Keep in mind we do not have access to your cluster or dags.
+
+--->
+
+**Anything else we need to know**:
+
+<!-- If this is actually about documentation, add `/kind documentation` below -->
```

Comment:
kind?

Reply:
Oh copy-paste left over from the project I copied these from.